### DISCUSSION TITLE
Replace emmet-ls / add ls_emmet?

### DISCUSSION THREAD

**Discussion by alex-popov-tech** [williamboman/nvim-lsp-installer#433] [Q&A] [answered]
Hello @williamboman , thank you for that beautiful piece of software again <3

My question is - how do you feel about adding new one/ replacing existing `emmet-ls` server?

Reasons:
- its not working or working incorrectly in some scenarios ( not working in JSX/TSX, and others here https://github.com/aca/emmet-ls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc )
- author seems to be absent for a while and stopped supporting `emmet-ls`

Propositions:
Add/replace existing with great forks supported by community, like  https://github.com/pedro757/emmet or https://github.com/kozer/emmet-language-server ( i tried both today, and for me first one works best ).

If you agree on having that one in your plugin, im more than glad to create PR for that <3

**Comment by williamboman** [comment, marked as answer]
Hello @alex-popov-tech and thank you 😊!! I'd be happy to change to a different server, if it's objectively more robust than the existing one. However, it's not uncommon to have multiple servers for the same "language", we have many such cases in this plugin. Also, this plugin follows what's provided by lspconfig, so I think a good first step is to bring this suggestion over there instead. 

I remember there were talks about which server implementation to use, I'll try to find the thread again.

**Reply by williamboman** [reply]
It was first added here, with some comments about which server to actually use https://github.com/neovim/nvim-lspconfig/pull/1295

**Reply by alex-popov-tech** [reply]
i was looking at original `emmet-ls` repo for some time, it looks light author is there sometimes, but support is not very rapid...
I won't be proposing new one then, and just in case will drop a link on how-to setup custom server like `ls_emmet` https://github.com/alex-popov-tech/.dotfiles/blob/4074ce8581a9151916f9dc56a62cd281e4b7de23/nvim/.config/nvim/lua/lsp/servers/customs.lua
thank you @williamboman <3
